### ISSUE TITLE
CNAME ERROR PgRouting

### ISSUE THREAD

**Issue by nedabalafkan** [pgRouting/pgrouting#724]
### Expected behavior and actual behavior
It was supposed to create a topology, however if the command is run an error "CNAME occurs. 

### Steps to reproduce the problem
-- Executing query:
SELECT  pgr_createTopology('edge_table',0.001);
NOTICE:  PROCESSING:
NOTICE:  pgr_createTopology('edge_table', 0.001, 'the_geom', 'id', 'source', 'target', rows_where := 'true', clean := f)
NOTICE:  Performing checks, please wait .....
NOTICE:  Unexpected error column "cname" does not exist
Total query runtime: 93 msec
1 row retrieved.

### Specifications like the version of pgRouting/PostGIS and PostgreSQL as well as Operating System

Use the commands:
```
SELECT version();  "PostgreSQL 9.5.5 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.2) 5.4.0 20160609, 64-bit"
SELECT postgis_full_version();  "POSTGIS="2.2.2 r14797" GEOS="3.5.0-CAPI-1.9.0 r4084" PROJ="Rel. 4.9.2, 08 September 2015" GDAL="GDAL 2.1.0, released 2016/04/25" LIBXML="2.9.3" LIBJSON="0.11.99" RASTER"
SELECT pgr_version();   "(2.2.3,pgrouting-2.2.3,6d05b9b,master,1.58.0)"
```
Postgis 2.2.2
plpgsql 1.0
pgrouting extension 2.2.3
Linux Ubuntu
pgadmin 3

**Comment by dkastl** [comment]
Do you run your command with a user account, that has limited privileges?

**Comment by nedabalafkan** [comment]
I'm not certain , but i presume it.   
Naida 
-- 
Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.On 13/01/2017, 02:23 Daniel Kastl <notifications@github.com> wrote:

  
    
      Do you run your command with a user account, that has limited privileges?

**Comment by dkastl** [comment]
Can you try with a user account such as `postgres` for example to make sure, that it's not a permission problem.

**Comment by nedabalafkan** [comment]
I have tried that
-- 
Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.On 13/01/2017, 03:09 Daniel Kastl <notifications@github.com> wrote:

  
    
      Can you try with a user account such as postgres for example to make sure, that it's not a permission problem.

**Comment by cvvergara** [comment]
I believe this is fixed on version 2.4-alpha
can you please try:
https://github.com/pgRouting/pgrouting/releases/tag/v2.4.0-alpha
